packages/opentelemetry-metrics/src/export/ConsoleMetricExporter.ts

```diff
@@ -36,11 +36,12 @@ export class ConsoleMetricExporter implements MetricExporter {
       console.log(metric.descriptor);
       console.log(metric.labels);
       switch (metric.descriptor.metricKind) {
-        case MetricKind.COUNTER:
+        case MetricKind.COUNTER: {
```

Comment:
Not sure how I feel about requiring this to be a block. I have never seen this syntax before even.

Reply:
Same for me too, but after reading this (https://eslint.org/docs/rules/no-case-declarations), it makes sense... WDYT ?

Reply:
its fine